query/src/configs/config_storage.rs

```diff
@@ -185,6 +198,10 @@ pub struct StorageConfig {
     // azure storage blob config.
     #[clap(flatten)]
     pub azblob: AzblobStorageConfig,
+
+    // hdfs storage backend config
+    #[clap(flatten)]
+    pub hdfs: HdfsConfig,
```

Comment:
If we don't add the `HdfsConfig` item to *.toml, the config deserialize looks will crash?

Reply:
This is another problem, we don't want unused config items explicitly:
```
[storage]
# fs|s3
type = "s3"

[storage.fs] -- this config

[storage.s3]
bucket = "databend"
endpoint_url = "https://s3.amazonaws.com"
access_key_id = "<your-key-id>"
secret_access_key = "<your-access-key>"

[storage.azblob] -- this config
```

Is it possible to configure the item only we used, like `storage.s3`?

Reply:
> If we don't add the `HdfsConfig` item to *.toml, the config deserialize looks will crash?

If `HdfsConfig` is not added in `*.toml`, we will use the default value instead.

I tested this behavior locally: `query` is able to start without adding hdfs-related staff.

Reply:
That's great, I am out, thank you.
I will remove all the unused config items from the documents.